### PR TITLE
feat: add decision logs to project

### PR DESCRIPTION
**Pull request by djaiss**
image

You fool, don't forget these steps:

- [x] Unit tests
- [x] Tests with Cypress
- [x] Documentation
- [x] Dummy data

### CODE REVIEW

File: app/Models/Company/ProjectDecision.php [hunk 1]
```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace App\Models\Company;
+
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\Relations\BelongsTo;
```
Comment (asbiin) [inline app/Models/Company/ProjectDecision.php:6]: You've missed some definition:
```suggestion
use Illuminate\Database\Eloquent\Relations\BelongsTo;
use Illuminate\Database\Eloquent\Relations\BelongsToMany;
```

**Review by asbiin — COMMENTED** [review]
See my comment

File: webpack.mix.js [hunk 2]
```diff
@@ -6,6 +6,8 @@ let purgeCssOptions = {
   enabled: true,
   // List of regex of CSS class to not remove
   whitelistPatterns: [/^ball-pulse/],
+  whitelistPatterns: [/^vs__/],
```
Comment (asbiin) [inline webpack.mix.js:9]: There is the reason ball-pulse was not working anymore:
```suggestion
  whitelistPatterns: [/^ball-pulse/, /^vs__/],
```

File: webpack.mix.js [hunk 3]
```diff
@@ -6,6 +6,8 @@ let purgeCssOptions = {
...
+  whitelistPatternsChildren: [/^vs-/],
   // List of regex of CSS class name whose child path CSS class will not be removed
   //  ex: to exclude "jane" in "mary jane": add "mary")
   whitelistPatternsChildren: []
```
Comment (asbiin) [inline webpack.mix.js:13]: Please fix it like this:
```suggestion
  // List of regex of CSS class name whose child path CSS class will not be removed
  //  ex: to exclude "jane" in "mary jane": add "mary")
  whitelistPatternsChildren: [/^vs-/],
```

**Review by asbiin — COMMENTED** [review]
We can close #330

File: app/Models/Company/ProjectDecision.php (hunk 1 above)
Comment (djaiss) [reply app/Models/Company/ProjectDecision.php:6]: Fixed

File: app/Helpers/LogHelper.php [hunk 4]
```diff
@@ -856,6 +856,22 @@ public static function processAuditLog(AuditLog $log): string
...
+                break;
+
+            case 'project_decision_destroyed':
+                $sentence = trans('account.log_project_decision_destroyed', [
+                    'project_id' => $log->object->{'project_id'},
+                    'project_name' => $log->object->{'project_name'},
+                    'title' => $log->object->{'title'},
+                ]);
+
+                // no break
```
[the remark this answers is not in the document]
Comment (djaiss) [reply app/Helpers/LogHelper.php:874]: oops - will fix in another PR

**Pull request merged**